subtrack.MAUI/Services/SubscriptionsCalculator.cs

```diff
@@ -49,6 +49,11 @@ public decimal GetYearlyCost(Subscription subscription)
         return subscription.Cost * yearlyPaymentsCount;
     }
 
+    public decimal GetAverageYearlyCost(Subscription subscription)
```

Comment:
This should replace the "GetYearlyCost" so you can remove the other method, easiest is to rename the other one to GetAverageYearlyCost so that all the callers have the method auto renamed. Add some tests (doesnt need to be many) aswell that covers:

interval of 1,2 and 3
All subscription billing types

Reply:
Oh okay. I thought you wanted both methods (actual cost and average cost) to exist. Not a problem.

Reply:
Need to get this done for next release since creating yearly subs causes an exception rn. Can chip in tomorrow and fix the remaining things